cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-deny-syntax.asciidoc

```diff
@@ -66,7 +66,7 @@ this privilege can only be assigned to all nodes, relationships and properties i
 *** `ELEMENTS` label (both nodes and relationships).
 ** The label or type can be `+*+` which means all labels or types.
 ** Multiple labels or types can be specified, comma-separated.
-** Defaults to `ELEMENTS +*+` if omitted.
+** Defaults to `ELEMENTS` `+*+` if omitted.
```

Comment:
Out of curiosity, can you explain what difference this leads to?

Reply:
This is how it looks in 4.1 and forward, it's to not actually print the `+` but use them for escaping the `*` so you don't get bold text, it will however look like '`ELEMENTS` `*`' instead of '`ELEMENTS *`' (but at least don't have `ELEMENTS +*+`)

Not sure if it is needed in this case but I just followed how it had been fixed in 4.1